package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1799,6 +1800,137 @@ def n_fragments(self):
         ('n_fragments', property(n_fragments, None, None,
                                  n_fragments.__doc__)))
 
+    def unwrap(self, box=None, compound='fragments', reference='com',
+               inplace=True):
+        """Move atoms in this group so that bonds within the
+        group's compounds aren't split accross periodic boundaries.
+
+        Parameters
+        ----------
+        box : array_like or None, optional
+            The unitcell dimensions of the system, which can be orthogonal or
+            triclinic and must be provided in the same format as returned by
+            :attr:`MDAnalysis.coordinates.base.Timestep.dimensions`:\n
+            ``[lx, ly, lz, alpha, beta, gamma]``.
+            If ``None``, uses the dimensions of the current timestep.
+        compound : {'group', 'segments', 'residues', 'molecules', 'fragments'}, optional
+            Which type of components to unwrap. Note that, in any case, all
+            atoms within each compound must be interconnected by bonds, i.e.,
+            compounds must correspond to (parts of) molecules.
+        reference : {'com', 'cog', None}, optional
+            If ``'com'`` (center of mass) or ``'cog'`` (center of geometry), the
+            unwrapped compounds will be shifted so that their individual
+            reference point lies within the primary unit cell.
+            If ``None``, no such shift is performed.
+        inplace : bool, optional
+            If ``True``, coordinates are modified in place.
+
+        Returns
+        -------
+        coords : numpy.ndarray
+            Unwrapped atom coordinate array of shape ``(n, 3)``.
+
+        Note
+        ----
+        * This is the inverse transformation of packing atoms into the primary
+          unit cell, preventing bonds from being "broken" accross periodic
+          boundaries.
+        * Be aware of the fact that only atoms *belonging to the group* will
```

Comment:
probably deserves an example of how to select the fragments atoms in an atomgroup belong to

Reply:
Is there a more elegant way than constructing a group containing all atoms of all fragments in `ag` than `ag_all_frag_atoms = sum(ag.fragments)`?

Reply:
that's fine, internally it should just be going through to `AtomGropu(np.concatenate([ag.ix for ag in ag.fragments]))` without too many layers